builtin/common/vector.lua

```diff
@@ -375,6 +375,24 @@ function vector.in_area(pos, min, max)
 		(pos.z >= min.z) and (pos.z <= max.z)
 end
 
+function vector.random_direction()
+	-- Generate a random direction of unit length, via rejection sampling
+	local x = math.random(-1,1)
```

Comment:
You made the same mistake as @kromka-chleba here: This returns a random integer from -1 to 1 (inclusive), not a random real number. You want `math.random() * 2 - 1` for a real number.

Reply:
Thank you, I am not a lua expert. Did not expect this, since lua does not have integers.